ProcessLib/PhaseField/PhaseFieldProcess-impl.h

```diff
@@ -143,7 +255,33 @@ void PhaseFieldProcess<DisplacementDim>::assembleConcreteProcess(
     DBUG("Assemble PhaseFieldProcess.");
 
     std::vector<std::reference_wrapper<NumLib::LocalToGlobalIndexMap>>
-       dof_tables = {std::ref(*_local_to_global_index_map)};
+        dof_tables;
+
+    // For the monolithic scheme
+    if (_use_monolithic_scheme)
+    {
+        DBUG("Assemble PhaseFieldProcess for the monolithic scheme.");
+        dof_tables.emplace_back(*_local_to_global_index_map);
+    }
+    else
+    {
+        // For the staggered scheme
+        if (_coupled_solutions->process_id == 0)
```

Comment:
Change the process orders as that in assembleWithJacobianConcreteProcess, thought the Picard scheme is not used in the current implementation.

Reply:
The process order is now consistent.